it/controllers/OrchestratorControllerISpec.scala

```diff
@@ -16,49 +16,135 @@
 
 package controllers
 
+import org.mongodb.scala.result.DeleteResult
+import org.mongodb.scala.bson.collection.immutable.Document
 import play.api.libs.json.{JsValue, Json}
 import play.api.test.Helpers.{await, _}
-import utils.{IntegrationSpecCommonBase, MockFileNotification}
+import utils.IntegrationSpecCommonBase
 import org.scalatest.matchers.should.Matchers._
+import play.api.libs.ws.WSResponse
+import repositories.FileNotificationRepository
 
-class OrchestratorControllerISpec extends IntegrationSpecCommonBase with MockFileNotification {
+import scala.concurrent.Future
+
+class OrchestratorControllerISpec extends IntegrationSpecCommonBase {
 
   val controller: OrchestratorController = injector.instanceOf[OrchestratorController]
+  lazy val repository: FileNotificationRepository = injector.instanceOf[FileNotificationRepository]
+
+  def deleteAll(): Future[DeleteResult] =
+    repository
+      .collection
+      .deleteMany(filter = Document())
+      .toFuture
+
+  class Setup {
+    await(deleteAll())
+  }
+
+  val jsonToReceive: JsValue = Json.parse(
+    """
+      |[{
+      |   "informationType": "type",
+      |   "files": {
+      |       "recipientOrSender": "recipient",
+      |       "name": "John Doe",
+      |       "location": "place",
+      |       "checksum": {
+      |           "algorithm": "beep",
+      |           "value": "abc"
+      |       },
+      |       "size": 1,
+      |       "properties": [
+      |       {
+      |           "name": "name",
+      |           "value": "xyz"
+      |       }]
+      |   },
+      |   "audit": {
+      |       "correlationID": "12345"
+      |   }
+      |}]
+      |""".stripMargin
+  )
+
+  "receiveSDESNotifications" should {
+    "call FileNotificationRepositories - returns OK" in new Setup {
+      val result: WSResponse = await(buildClientForRequestToApp(uri = "/new-notifications").post(
+        jsonToReceive
+      ))
+      result.status shouldBe OK
```

Comment:
You could check to see if the repository has stored the data

Reply:
Could you check this?

Reply:
Have added a check for this